src/gtk/dialog.cpp

```diff
@@ -163,7 +162,7 @@ int wxDialog::ShowModal()
     gtk_window_set_modal(GTK_WINDOW(m_widget), true);
 
     m_modalShowing = true;
-	
+
```

Comment:
Changes to blank spaces, like the ones in this file, should have been discarded. But I guess they are easy to remove in the actual commit.